src/services/dashboards/dashboard-detail/store/dashboard-detail-info.ts

```diff
@@ -58,8 +56,8 @@ interface DashboardDetailInfoStoreState {
 }
 interface ValidationState {
     isNameValid?: boolean;
-    isWidgetLayoutValid: ComputedRef<Record<string, boolean>>;
-    widgetInheritVariablesValidMap: ComputedRef<WidgetInheritVariablesValidMap>;
+    isWidgetLayoutValid: ComputedRef<boolean>;
+    widgetInheritVariablesValidMap: WidgetInheritVariablesValidMap;
```

Comment:
Question: Does the validation of the form not include? If it's included, I think we need to modify the variable name..!